### ISSUE TITLE
`all` property is misplaced during initial formatting

### ISSUE THREAD

**Issue by Birdie0** [theKashey/stylelint-semantic-groups#9]
Using stylelint-semantic-groups with [recommended config](https://github.com/theKashey/stylelint-semantic-groups#recommended-configuration) on next css snippet

```css
.btn {
  all: unset;
  border: 1px solid #639;
  color: #639;
  cursor: pointer;
  padding: 5px;
}
```

turns it into

```css
.btn {
  padding: 5px;

  color: #639;

  border: 1px solid #639;

  cursor: pointer;
  all: unset;
}
```

Resulting CSS will render matched element unstyled as `all: unset` overrides all default and previously specified CSS properties.

**Comment by theKashey** [comment]
👍  all "unknown" properties are places at the end and all is unknown.
Look like it's should be a very first one instead.

**Comment by theKashey** [comment]
Thank you for the report. Solved at stylelint-semantic-groups@1.2.1
